app/code/Magento/Sales/Api/Data/OrderInterface.php

```diff
@@ -580,6 +580,7 @@ public function getAdjustmentPositive();
 
     /**
      * Gets the applied rule IDs for the order.
+     * Rules are comma separated if there are more than one.
```

Comment:
Please include a blank line before or add the "(comma separated)" information in return line, removing this one.